logger.js

```diff
@@ -82,45 +95,45 @@ Logger.prototype.destroy = function destroy() {
     }, this);
 };
 
-module.exports = Logger;
-
-function logMethod(logger, levelName, level, path) {
-    var streams = logger.streams;
-    var logStreams = level.backends.reduce(function (acc, name) {
-        if (!streams[name]) {
-            return acc;
-        }
-
-        acc[name] = streams[name];
-        return acc;
-    }, {});
+Logger.prototype.log = function log(entry, callback) {
```

Comment:
I changed log to writeEntry since this method is public or private depending on the perspective of the actor.